app/mailers/devise_user_mailer.rb

```diff
@@ -16,4 +17,12 @@ def confirmation_instructions(record, token, opts = {})
     @prefill_token = opts[:prefill_token]
     super
   end
+
+  def add_delivery_method
+    headers[BalancerDeliveryMethod::FORCE_DELIVERY_METHOD_HEADER] = SafeMailer.forced_delivery_method
+  end
+
+  def forced_delivery?
+    SafeMailer&.forced_delivery_method.present?
```

Comment:
```suggestion
    SafeMailer.forced_delivery_method.present?
```

C'est une méthode de classe, elle sera toujours accessible